test/stability-tests/README.md

```diff
@@ -0,0 +1,13 @@
+# SDK Stability Regression Tests
+
+## Description
```

Comment:
Please add a description of what 'stability' means, and also describe the current tests